src/pages/profile/index.tsx

```diff
@@ -0,0 +1,158 @@
+import Box from '@mui/material/Box';
```

Comment:
use shared/ui for reexport everything related ui kit

Reply:
fixed